docs/source/index.mdx

```diff
@@ -1,10 +1,14 @@
 ---
 title: Introduction to Apollo iOS
 sidebar_title: Introduction
-description: A strongly-typed, caching GraphQL client for iOS, written in Swift
+description: A Strongly typed, Swift-first, GraphQL client
 ---
 
-**Apollo iOS** is an [open-source](https://github.com/apollographql/apollo-ios) GraphQL client for native iOS apps, written in Swift. It enables you to execute queries and mutations against a GraphQL server and returns results as operation-specific Swift types.
+[**Apollo iOS**](https://github.com/apollographql/apollo-ios) is an open-source GraphQL client for native client applications, written in Swift.
```

Comment:
I recommend keeping the link on "open-source", as jumping out to the repo at this point is presumably for someone to actually check out the source. Alternatively, we could cut this link entirely to prevent immediate dropoff